browser/src/map/Clipboard.js

```diff
@@ -857,10 +857,6 @@ L.Clipboard = L.Class.extend({
 	// Executes the navigator.clipboard.write() call, if it's available.
 	_navigatorClipboardWrite: function() {
 		if (!L.Browser.clipboardApiAvailable) {
-			// Show a visible warning, this should not happen in production.
-			this._map.uiManager.showSnackbar(
-				_('The async Clipboard API is not supported by your browser, switching to HTTPS is meant to fix that.')
-			);
```

Comment:
OK. I think initially @eszkadev wanted this, because first it was not clear that http:// gives no new/async clipboard, even on the latest Chrome. But if the warning causes more problems than it solves, then let's forget about it.